hack/get_aihub_manifests.sh

```
@@ -0,0 +1,145 @@
+        if [[ $arg == --* ]]; then
+            arg="${arg:2}"
+            IFS="=" read -r key value <<< "$arg"
+            if [[ -n "${COMPONENT_MANIFESTS[$key]}" ]]; then
+                if [[ ! $value =~ $pattern ]]; then
+                    echo "ERROR: The value '$value' does not match the expected format 'repo-org:repo-name:ref-name:source-folder'."
+                    continue
+                fi
+                COMPONENT_MANIFESTS["$key"]=$value
+            else
+                echo "ERROR: '$key' does not exist in COMPONENT_MANIFESTS, it will be skipped."
+                echo "Available components are: [${!COMPONENT_MANIFESTS[@]}]"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)